javascript/javascript2/week3/task3.js

```diff
@@ -0,0 +1,88 @@
+
+
+
+const refs = {
+    input: document.querySelector('[data-input=value]'),
+    button: document.querySelector('.btn-start'),
+    outputS: document.querySelector('.outputS'),
+    outputL: document.querySelector('.outputL'),
+    timeOver: document.querySelector('.timer-over'),
+    divS: document.querySelector('.gamer-s'),
+    divL: document.querySelector('.gamer-l'),
+    
+}
+let inputValue = 0;
+let counterS = 0;
+let counterL = 0;
+
+refs.button.addEventListener('click', onClickBtnStart);
+document.addEventListener('keyup', onKeyupCounter)
+
+
+
+function onClickBtnStart(event) {
+    inputValue = refs.input.value;
+    if (inputValue === "") {
+       return refs.timeOver.textContent = 'Set timeout'
+    }
+    const timeoutId = setTimeout( defineWinner,
+        inputValue * 1000)
+        
+        if (inputValue !== "") {
```

Comment:
You do not need this if statement here. The first if statement in this funcion has a return keyword. That means id input value will ve empty we will exit this funcion on line 26 and any code we have below will not be executed. So we can be sure that if we ever get to line 31 of this funcion the input value will not be empty string and we can remove the second if statemet. 
You did some really good work here with handling edge cases in your code by adding it at the begingng of the funcion and exiting it with `return`. Good work!